src/main/java/domain/piece/Bishop.java

```diff
@@ -0,0 +1,35 @@
+package domain.piece;
+
+import domain.position.Position;
+
+public class Bishop extends AbstractPiece {
+    public Bishop(Color color) {
+        super(color);
+    }
+
+    @Override
+    public void validateMovement(Position resource, Position target, Piece other) {
+        if (this.getColor() == other.getColor()) {
+            throw new IllegalArgumentException("같은 팀의 말을 잡을 수 없습니다.");
+        }
+        int rankGap = resource.calculateRankGap(target);
+        int fileGap = resource.calculateFileGap(target);
+
+        if (rankGap == 0 && fileGap == 0) {
+            throw new IllegalArgumentException(String.format("%s은 대각선 방향으로만 이동할 수 있습니다.",
+                    this.getClass().getSimpleName()));
+        }
+
+        if (rankGap == fileGap) {
+            return;
+        }
+
+        throw new IllegalArgumentException(String.format("%s은 대각선 방향으로만 이동할 수 있습니다.",
+                this.getClass().getSimpleName()));
```

Comment:
```suggestion
        if (rankGap != fileGap) {
            throw new IllegalArgumentException(String.format(
                "%s은 대각선 방향으로만 이동할 수 있습니다.",
                this.getClass().getSimpleName()));
        }
```
예외 상황때 예외를 발생시키고 return 을 나중에 하는 형태를 유지해주세요.
그리고 줄바꿈을 적절히 활용하여 가독성을 높여보는 것도 괜찮을 것 같네요!

Reply:
반영했습니다!